### ISSUE TITLE
Deployment Failed. deployer = GITHUB_ZIP_DEPLOY deploymentPath = ZipDeploy. Extract zip. Remote build. Error: Failed to deploy web package to App Service. Error: Deployment Failed, Package deployment using ZIP Deploy failed. Refer logs for more details.

### ISSUE THREAD

**Issue by Samirit** [Azure/webapps-deploy#362]
My github web app is getting successfully built but everytime I try to deploy it to azure, I get the following error:

Deployment Failed. deployer = GITHUB_ZIP_DEPLOY deploymentPath = ZipDeploy. Extract zip. Remote build.
Error: Failed to deploy web package to App Service.
Error: Deployment Failed, Package deployment using ZIP Deploy failed. Refer logs for more details.

I'm not getting any information about why it's not getting deployed in the logs either. What should I do?

**Comment by sasha-bachurin** [comment]
+1 same issue

**Comment by KalvinWei** [comment]
@Samirit Hi, mate. I recently experienced some failure but it's quite weird.
I have 2 deployment slots, one for production and one for staging. Accordingly, I have 2 action YAMLs to deploy to the 2 slots respectively. The thing is the deployment to the production slot always succeeded without any issue, meanwhile the deployment to the staging slot always stuck in the Deploy to Azure WebApp phase and ran for hours and then failed. BUT, actually the deployment was indeed successfully if I browser the web app and view the deployment log in Azure Console.

So, I wonder it's in fact deployed successfully, but the deployment action was not signalled the success so it stuck there and then exceeded the GitHub Action runner's timeout limit.

Do you experience alike?

**Comment by beelol** [comment]
Same issue here, using out of the box azure deploy script from github with ubuntu, node 18

**Comment by anish-dev-127** [comment]
same error not able to deploy nextjs 13 on azure

**Comment by guilhermescard** [comment]
I'm having the same problem intermittently when deploying .NET

**Comment by jelkhoury** [comment]
same for me, it started a week ago when i scaled the app service plan to 2 instances. any workaround ?

**Comment by klinki** [comment]
Same issue here

**Comment by ashwath007** [comment]
Same issue , any updates ?

**Comment by dominon12** [comment]
Same issue over and over...

**Comment by Namle-git** [comment]
I'm having the same issue. Seems like torch is what causing the problem for me. The app will successfully deploy with WEBSITE_RUN_FROM_PACKAGE set to 1 but the app will not run. Everything runs fine if torch is removed from requirements.txt

**Comment by Piedone** [comment]
Correct: deployment fails with this every single time, but if I re-run it quickly, then it'll succeed.
